templates/bundles/PageonDoctrineDataGridBundle/dataGrid.html.twig

```diff
@@ -1,40 +1,38 @@
-{% import '@Backend/base/macros.html.twig' as macro %}
-
-{% set dataGridEmptyAction = dataGridEmptyAction|default(null) %}
-{% set dataGridEmptyModule = dataGridEmptyModule|default(null) %}
-{% set dataGridEmptyParameters = dataGridEmptyParameters|default([]) %}
-{% set dataGridEmptyRelative = dataGridEmptyRelative|default(false) %}
-{% set dataGridEmptyLocale = dataGridEmptyLocale|default(null) %}
-{% set dataGridEmptyLabel = dataGridEmptyLabel|default('lbl.Add'|trans|ucfirst) %}
-{% set dataGridWrapperClass = dataGridWrapperClass|default('table-responsive fork-data-grid') %}
+{% set data_grid_empty_action = dataGridEmptyAction|default(null) %}
```

Comment:
Mag je deze variabelen zomaar aanpassen? Want die kunnen toch in andere templates van de bundle verwacht worden?

Reply:
De set variable wordt hier lokaal gebruikt de meegeven variable is niet veranderd